### PR TITLE
21-redirect-landing-after-deleting-user

### PR DESCRIPTION
**Pull request by CarlosACJ55**
## Rationale
User deletion was still left in the past, with no htmx or routing enabled.

## Usage
Delete a user, and voila! you are now on the landing page and logged out with your cookie invalidated/

### Non-trivial Files
N/A

### Testing
- [ ] ➕ Added units to existing tests
- [ ] 🐣 Started a new group of tests
- [ ] 🙅 New tests aren't needed
- [x] 🙋 Need help to add tests

### Dependencies
N/A

### Documentation Changes
- [ ] 📜 README.md
- [ ] 💬 Added **necessary** comments to existing code
- [ ] 📕 storybook
- [x] 🙅 no unrelated documentation changes

## Issues and Bugs
Need integration testing to test anything here

## Possible Solutions
Create integration tests in the future

## Additional Notes
N/A

### CODE REVIEW

File: handlers.go [hunk 1]
```diff
@@ -22,11 +22,11 @@ var (
 	surfacePages = map[string]struct{}{"signup": {}, "login": {}}
 )
 
-func servePublicFile(w http.ResponseWriter, r *http.Request) {
+func getPublic(w http.ResponseWriter, r *http.Request) {
```
Comment (Alishah634) [inline handlers.go:25]: Could you explain why you decided to change the name for this function?

Comment (CarlosACJ55) [reply handlers.go:25]: I decided to make all the function names for handlers very clear. get[RequestName] for GET requests, and post[RequestName] for POST requests

**Review by Alishah634 — APPROVED** [review]
LGTM

File: handlers.go (hunk 1 above)
Comment (Alishah634) [reply handlers.go:25]: Okay, if this is the format from now on then please update documentation to reflect this as a standard for the code base.

Comment (CarlosACJ55) [reply handlers.go:25]: Where would you put this? Its a file full of url request handlers, so its kind of in context that the requests can be of different kinds.

**Pull request merged**